obj/objects.h

```diff
@@ -59,18 +59,18 @@ namespace objects
     class SphereWall : public Sphere
     {
     public:
+        SphereWall(double radius = constants::kDefaultSimulationRadius) { setRadius(radius); }
```

Comment:
Drop the default value, this way it will be easier to get rid of this dependency.

You can maybe do it in a future commit as this is becoming quite large already.

Reply:
done